packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml

```diff
@@ -16,6 +16,12 @@ spec:
       Tag: "eks"
       Path: "/var/log/messages"
       Parser: "syslog"
+    {{ if .Values.fluentbit_tolerations }}
```

Comment:
I think this `if` is redundant? same for the other places

Reply:
good catch.